docker/app
# Fix shorthand help flag

**- What I did**
Bump the docker/cli vendoring so that the shorthand help flag (`-h`) works. See: https://github.com/docker/cli/pull/1787.

This also required bumping docker/docker.

**- How I did it**
```console
$ dep ensure -update github.com/docker/cli
$ dep ensure -update github.com/docker/docker
```
**- How to verify it**

See vendoring changes in `Gopkg.lock`.

Using Docker App plugin:
```console
$ docker app bundle -h
```

Should see output for `bundle` command.

**- Description for the changelog**
None

**- A picture of a cute animal (not mandatory but encouraged)**
